config.go

```diff
@@ -1790,6 +1796,13 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 		)
 	}
 
+	// Validate that the node isn't configured as both a remote signer and a
+	// watch-only node.
+	if cfg.RemoteSigner.Enable && cfg.WatchOnlyNode.Enable {
+		return nil, fmt.Errorf("cannot enable both the remotesigner " +
```

Comment:
nit: `cannot be configured as both a watchonly node and a remote signer node` - feels a bit clearer